ogcapi-features/README.md

```diff
@@ -97,13 +104,17 @@ The following Link relations must exist in the Landing Page (root).
 | `conformance` | `/conformance` | OAFeat   | Conformance URI     |
 | `data`        | `/collections` | OAFeat   | List of Collections |
 
+### Collections (/collections)
+
 The following Link relations must exist in the `/collections` endpoint response.
 
 | **rel** | **href**       | **From**  | **Description** |
 | ------- | -------------- | --------- | --------------- |
 | `root`  | `/`            | STAC Core | The root URI    |
```

Comment:
This requirement can't really come from STAC Core as a Collections list does not exist in STAC Core.

Reply:
Agree about From not being Core -- I'll fix that.

Same for /items -- the root Catalog is the parent of all the Collections, but not the "CollectionCollection" returned by /collections, so we don't use parent here. Presumably we could add the equivalent of the `collections` link rel in /items, but it's always going to be the same as root (I think?) and I don't know what link rel name would be -- catalog?